manual/manual/tutorials/objectexamples.etex

```diff
@@ -2,15 +2,20 @@
 \label{c:objectexamples}
 \pdfchapterfold{-15}{Tutorial: Objects in OCaml}
 %HEVEA\cutname{objectexamples.html}
-{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
+{\it (Chapter written by J�r�me Vouillon, Didier R�my and Jacques Garrigue)}
```

Comment:
Editor encoding artefact?

Reply:
My local repository doesn't show any difference here, @Octachron thought it might be a github diff-code glitch because of the character encoding (ISO 8859-1 single byte vs UTF-8).

Reply:
A safer solution might be to use `(Chapter written by J\'er\^ome Vouillon, Didier R\'emy and Jacques Garrigue)` and then we could discuss later to change the encoding of the manual to utf-8.

Reply:
I applied @Octachron 's suggestion.

Reply:
Note quite correctly, though - the letters themselves are missing after the accents. J`\'er\^ome` and `R\'emy`

Reply:
Doh.  Fixed.